src/inspect_ai/_view/www/dist/assets/index.js

```diff
@@ -16527,12 +16536,50 @@ const ScoreEventView = ({ id, event, style }) => {
 
   </${EventPanel}>`;
 };
+const ApprovalEventView = ({ id, event, style }) => {
+  return m$1`
+  <${EventRow}
+      id=${id}
+      title="${decisionLabel(event.decision)}"
+      icon=${decisionIcon(event.decision)}  
+      style=${style}
+    >
+    ${event.explanation}
+  </${EventRow}>`;
+};
+const decisionLabel = (decision) => {
+  switch (decision) {
+    case "approve":
+      return "Approved";
+    case "reject":
+      return "Rejected";
+    case "terminate":
+      return "Terminated";
+    default:
+      return decision;
+  }
+};
+const decisionIcon = (decision) => {
```

Comment:
There are two other possible decisions: 

- Modify (approved with modification of the tool call, which will be reflected in the tool event you are already displaying)
- Escalate (move to the next approver, in this case you'll see two approval events in the stream)